api/rds.proto

```diff
@@ -279,7 +279,12 @@ message VirtualCluster {
 
   // Optionally specifies the HTTP method to match on. For example GET, PUT,
   // etc.
-  string method = 3;
+  RequestMethod method = 3;
+
+  message DeprecatedV1 {
```

Comment:
I'll fire off a PR to remove this in the main repo without the official deprecation process, if everyone is OK with it, let's do that.